### PR TITLE
Remove not used interfaces

### PR DESCRIPTION
**Pull request by kant2002**
- IOmNavigator and IHTMLScreen do not used
- IMallocSpy belongs to test project and as such moved there

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6866)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/tests/TestUtilities/IMallocSpy.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Primitives/tests/TestUtilities/IMallocSpy.cs:1]: if this is used only in tests, why/what are the tests for?

Comment (kant2002) [reply src/System.Windows.Forms.Primitives/tests/TestUtilities/IMallocSpy.cs:1]: For tracking memory allocations. Follow the link for locations where actual usage happens https://github.com/dotnet/winforms/search?q=MallocSpyScope

**Comment by dreddy-work** [comment]
👍

File: src/System.Windows.Forms.Primitives/tests/TestUtilities/IMallocSpy.cs (hunk 1 above)
Comment (RussKie) [reply src/System.Windows.Forms.Primitives/tests/TestUtilities/IMallocSpy.cs:1]: It was added in https://github.com/dotnet/winforms/pull/4554.

**Review by RussKie — APPROVED** [review]

**Pull request merged**